schema/archive.json

```diff
@@ -0,0 +1,210 @@
+{
+  "collections": {
+    "archives": {
```

Comment:
This is just in case we need to handle multiple collection schema in that file as it was the case for the AMO blocklists. Yes a README would be useful.